build_resolvers/test/flutter_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Fast fail the script on failures.
+set -ex
```

Comment:
There are no `select`, `case`, or `for` here. I'm not sure why we'd want the extra verbosity if there were... Why do we need the `x` option?

Reply:
was just copy/pasting the header from other scripts, I can remove it though. The `-e` is the only part that really matters certainly.